tags/resources/toc-supporting-guides/general-technical-questions.md

```diff
@@ -0,0 +1,189 @@
+  * Explain how upgrades and rollbacks were tested and how the upgrade-\>downgrade-\>upgrade path was tested.  
+  * Explain how the project informs users of deprecations and removals of features and APIs.  
+  * Explain how the project permits utilization of alpha and beta capabilities as part of a rollout.
+
+## Day 2 \- Day-to-Day Operations Phase
+
+### Scalability/Reliability
+
+  * Describe how the project increases the size or count of existing API objects?  
+  * Describe how the project defines Service Level Objectives (SLOs) and Service Level Indicators (SLIs).  
+  * Describe any operations that will increase in time covered by existing SLIs/SLOs.  
+  * Describe the increase in resource usage in any components as a result of enabling this project, to include CPU, Memory, Storage, Throughput.  
+  * Describe which conditions enabling / using this project would result in resource exhaustion of some node resources (PIDs, sockets, inodes, etc.)  
+  * Describe the load testing that has been performed on the project and the results.  
+  * Describe the recommended limits of users, requests, system resources, etc. and how they were obtained.  
+  * Describe which resilience pattern the project uses and how, including the circuit breaker pattern.
+
+### Observability Requirements
+
+  * Describe the signals the project is using or producing, including logs, metrics, profiles and traces. Please include supported formats, recommended configurations and data storage.  
+  * Describe how the project captures audit logging.  
+  * Describe any dashboards the project uses or implements as well as any dashboard requirements.  
+  * Describe how the project surfaces project resource requirements for adopters to monitor cloud and infrastructure costs, e.g. FinOps  
+  * Which parameters is the project covering to ensure the health of the application/service and its workloads?  
+  * How can an operator determine if the project is in use by workloads?  
+  * How can someone using this project know that it is working for their instance?  
+  * Describe the SLOs (Service Level Objectives) for this project?  
+  * What are the SLIs (Service Level Indicators) an operator can use to determine the health of the service?
+
+### Dependencies
+
+  * Describe the specific running services the project depends on in the cluster.  
+  * Describe the project’s dependency lifecycle policy.  
+  * How does the project incorporate and consider source composition analysis as part of its development and security hygiene? Describe how this source composition analysis (SCA) is tracked.?   
+  * Describe how the project implements changes based on source composition analysis (SCA) and the timescale? 
+
+### Troubleshooting
+
+  * How does this project recover if a key component or feature becomes unavailable? e.g Kubernetes API server, etcd, database, leader node, etc.  
+  * Describe the known failure modes?
+
+### Security
```

Comment:
```suggestion
### Team Security
```

Reply:
seems this is more focused on the people than the section above, but could also be combined?

Reply:
This is security for Day 2, above was for Day 1

Reply:
updated if you like the suggestion

Reply:
@angellk does @xmulligan suggested change need incorporated to make the above more clear?

Reply:
Thank you for the suggestion @xmulligan ! 

Line 145 specifies these sub-sections are for Day 2. Would it make it more clear to have each sub-section say Day 2? 
E.g. Day 2 Observability, Day 2 Scalability, etc. - and then the same for Day 0 and Day 1 sub-sections?

I have been thinking through how best to incorporate your suggestion but am unsure at this point.

Reply:
ahh ok, I think I missed that before and security is the only section that is repeated from one to the next. I think it is also fine as is then

Reply:
Thanks @xmulligan !